app/assets/stylesheets/govuk_publishing_components/components/_intervention.scss

```diff
@@ -1,18 +1,29 @@
 .gem-c-intervention {
   @include govuk-text-colour;
-  @include govuk-responsive-padding(6, "left");
-  @include govuk-responsive-padding(6, "right");
-  @include govuk-responsive-padding(4, "top");
-  @include govuk-responsive-padding(4, "bottom");
+  @include govuk-responsive-padding(3);
   @include govuk-responsive-margin(6, "bottom");
   background-color: govuk-colour("light-grey", $legacy: "grey-4");
-
-  .gem-c-intervention__title {
-    @include govuk-responsive-margin(0);
-    @include govuk-responsive-padding(4, "bottom");
-  }
+  border-left: 10px solid $govuk-success-colour;
 
   .govuk-body:last-of-type {
     @include govuk-responsive-margin(0, "bottom");
   }
 }
+
+.gem-c-intervention__suggestion-link {
+  font-weight: 700;
+}
+
+.gem-c-intervention__dismiss {
+  @include govuk-responsive-margin(0, "bottom");
+  border: none;
+  font-weight: 700;
+  text-decoration: none;
```

Comment:
Because the idea is to eventually have the dismiss link be a button, that will look the same but be an actual `<button>` with the JS that goes with it. But for simplicity with this experiment, we're keeping as a link for now.
If that's not valid, I'm happy to bring it back to our designer. Let me know.

Reply:
I did add an underline on hover, though. Given it's currently a govuk-link I don't think it's particularly difficult, and it seems to be working in most browsers.

Reply:
I'd caution against building in anticipation of things that aren't here yet - also if the intention is for this to be a button that looks like a link that's even more of a reason for it to actually look like a link. But that's all not important for now.

Thanks for adding the underline on hover, unfortunately I think you need to turn it off when the element is also focussed, otherwise you get this:

![Screenshot 2021-10-05 at 09 50 43](https://user-images.githubusercontent.com/861310/135991694-bb561842-bd29-43d3-8f05-1627a60e4758.png)

Reply:
ok, makes sense to me (probably not to my designer but I can argue for making a button later).
You shouldn't see the underline anymore as a result.